README.md

```diff
@@ -34,6 +34,10 @@ Use terminal, iTerm, etc. Not XTerm.
         - _**With**_ remote user specified
             - `sh runAsUser.sh bjensen`
                 - `bjensen` is a user included in the test data.  Others are available.  For the purposes of the HWF event, use only known users.
+        - _**With gunicorn**_ and remote user specified
+            - `REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application`
```

Comment:
The specific user in the build is specifically for test. It is the reason why we are implementing Shib.